dev/archetypes/maven-java/src/main/resources/archetype-resources/pom.xml

```diff
@@ -69,55 +69,16 @@
         <dependencies>
             <dependency>
                 <groupId>com.lightbend.lagom</groupId>
-                <artifactId>lagom-javadsl-server_2.11</artifactId>
-                <version>${lagom.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>com.typesafe.play</groupId>
-                <artifactId>play-netty-server_2.11</artifactId>
-                <version>${play.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>com.lightbend.lagom</groupId>
-                <artifactId>lagom-javadsl-api_2.11</artifactId>
-                <version>${lagom.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>com.lightbend.lagom</groupId>
-                <artifactId>lagom-javadsl-persistence-cassandra_2.11</artifactId>
-                <version>${lagom.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>com.lightbend.lagom</groupId>
-                <artifactId>lagom-logback_2.11</artifactId>
-                <version>${lagom.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>com.lightbend.lagom</groupId>
-                <artifactId>lagom-javadsl-testkit_2.11</artifactId>
-                <version>${lagom.version}</version>
-                <scope>test</scope>
-            </dependency>
-            <dependency>
-                <groupId>com.lightbend.lagom</groupId>
-                <artifactId>lagom-javadsl-integration-client_2.11</artifactId>
+                <artifactId>lagom-maven-dependencies</artifactId>
                 <version>${lagom.version}</version>
+                <scope>import</scope>
+                <type>pom</type>
             </dependency>
             <dependency>
                 <groupId>com.typesafe.conductr</groupId>
                 <artifactId>${conductr.lib.name}</artifactId>
                 <version>1.4.7</version>
             </dependency>
-            <dependency>
-                <groupId>com.typesafe.akka</groupId>
-                <artifactId>akka-actor_2.11</artifactId>
-                <version>${akka.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>com.typesafe.akka</groupId>
-                <artifactId>akka-stream_2.11</artifactId>
-                <version>${akka.version}</version>
```

Comment:
Along with the `PLAY-VERSION` and `AKKA-VERSION` `archetypeVariables` in `build.sbt`.